package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Emit latest event to new observers",
   "main": "dist/hold.js",
+  "typings": "type-definitions/hold.d.ts",
```

Comment:
See [my comment in the @most/hold PR](https://github.com/mostjs/dom-event/pull/3/files#r50641195)